extensions/2.0/Khronos/KHR_materials_iridescence/README.md

```diff
@@ -0,0 +1,429 @@
+# KHR\_materials\_iridescence
+
+## Khronos 3D Formats Working Group
+
+* Pascal Schoen, adidas
+* Ben Houston, threekit
+* Mathias Kanzler, UX3D
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+
+Copyright (C) 2018-2021 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
```

Comment:
The PBR TSG held a poll on yesterday's call, and decided to move this extension from "Draft" to "Release Candidate" status.  This is a pre-ratification step that indicates to early adopters that the group believes the design has stabilized enough for the first set of implementations to be developed.

```suggestion
Release Candidate
```